src/scripts/component/master.js

```diff
@@ -76,38 +69,39 @@ let component = {
          *
          * @event click
          * @param {Object} e event
-         * @description simple event example
+         * @desc simple event example
          *
          */
-        this.$element.on('click', 'dt', (e) => {
+        this.element.addEventListener('click', (e) => {
```

Comment:
since we removed `$` from all examples, lets double check to make sure its not mentioned in the docs